lifechanger/src/app/experience-info/experience-info.component.css

```diff
@@ -0,0 +1,60 @@
+.experience-container {
+  height: 75px;
+}
```

Comment:
here is the same - check if it can be switched to dynamic (or just remove it) + BEM re-naming